app/js/main.js

```diff
@@ -185,7 +200,7 @@ function setupMeteringDemo(subscriptions) {
   // Handle clicks on the Metering Toast's "Subscribe" button.
   subscriptions.setOnNativeSubscribeRequest(() => {
     // Show a publisher paywall for demo purposes.
-    startFlow("showOffers");
+    startFlow('showOffers');
```

Comment:
I'm curious about several formatting changes in this PR. Which formatter did you use?

Reply:
swg-js uses `prettier`, so I think we should use `prettier` in this repository as well for consistency. I could work on formalizing that in code in a follow-up PR. For this PR though, I think `prettier` should still handle formatting for consistency and minimization of extra code changes

Reply:
I'm digging into this more, and realizing `prettier` also makes a number of changes to this file. For now, it'd be great if this PR could avoid formatting lines unrelated to the changes the PR aims to make

Reply:
I created a PR that gives Scenic the same formatting as `swg-js` - that should simplify this thread